lang/cs/Org.Apache.REEF.Network/Group/Task/Impl/CommunicationGroupNetworkObserver.cs

```diff
@@ -109,7 +109,7 @@ private IObserver<GroupCommunicationMessage> GetOperatorHandler(string operatorN
             {
                 if (!_handlers.ContainsKey(operatorName))
                 {
-                    LOGGER.Log(Level.Info, "handler for operator {0} has not been registered." + operatorName);
+                    LOGGER.Log(Level.Warning, "Handler for operator {0} has not been registered. You should never reach here!!!" + operatorName);
```

Comment:
This sounds less like a warning and more like an error. Or am I misunderstanding the context?